__tests__/app.test.js

```diff
@@ -15,3 +15,20 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics.length).toBe(3);
+        expect(topics[0]).toEqual({
+          slug: expect.any(String),
+          description: expect.any(String)
+        });
```

Comment:
Instead of checking topics[0] - use a forEach here. If you do topics.forEach((topic) then you can just replace line 26  with `topic` and it will check that every topic has a slug and description. 

The rest of this looks good!